### DISCUSSION TITLE
Windows 11 23H2 supported ?

### DISCUSSION THREAD

**Discussion by mpoirier101** [valinet/ExplorerPatcher#2510] [Q&A]
Like the title says, can we upgrade to Win11 23H2 and still use EP successfully ?
Thanks !

**Comment by tallman69** [comment]
Yes, The latest release, 22621.2506.60.1, has been tested on 23H2.  I have been using it on my three windows 11 computers running build 22631.2715 without any issues for me.

**Comment by as232** [comment]
I'm seeing issues with 23H2 and build 2506.60.1 using multiple virtual desktops. Taskbar icons will start disappearing (the application still available when using ALT+TAB, but no icon in the Taskbar.) Also seeing an "invisible" icon (no actual icon but a hilighted box aorund it and an underline indicating that it's got an open window.)

Restarting Explorer "fixes" this... until it happens again.

**Reply by pyrates999** [reply]
File that under a new issue then please.

**Comment by mpoirier101** [comment]
Sounds good, I will upgrade to 23H2 then... ( cross fingers... )

Thanks !

**Comment by mpoirier101** [comment]
Success !
Everything seems to work fine...  ( Single 4K Monitor, no virtual Desktops )

<img width="387" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/21262107/694db2f0-2b41-4449-aa46-5322c0394625">
